tests/View/ViewComponentAttributeBagTest.php

```diff
@@ -12,9 +12,11 @@ public function testAttributeRetrieval()
         $bag = new ComponentAttributeBag(['class' => 'font-bold', 'name' => 'test']);
 
         $this->assertSame('class="mt-4 font-bold" name="test"', (string) $bag->merge(['class' => 'mt-4']));
-        $this->assertSame('class="mt-4 font-bold" name="foo"', (string) $bag->merge(['class' => 'mt-4', 'name' => 'foo']));
+        $this->assertSame('class="mt-4 font-bold" name="test"', (string) $bag->merge(['class' => 'mt-4', 'name' => 'foo']));
+        $this->assertSame('class="mt-4 font-bold" id="bar" name="test"', (string) $bag->merge(['class' => 'mt-4', 'id' => 'bar']));
         $this->assertSame('class="mt-4 font-bold" name="test"', (string) $bag(['class' => 'mt-4']));
         $this->assertSame('class="mt-4 font-bold"', (string) $bag->only('class')->merge(['class' => 'mt-4']));
+        $this->assertSame('class="italic" name="test"', (string) $bag->only('name')->merge(['class' => 'italic', 'name' => 'default']));
```

Comment:
Nitpick: I would probably leave out the class in this assertion. That makes it more obvious that it's about testing the default handling, nothing else.

`$this->assertSame(name="test"', (string) $bag->only('name')->merge(['name' => 'default']));`